app/javascript/channels/room_channel.js

```diff
@@ -0,0 +1,33 @@
+import consumer from "./consumer"
+
+document.addEventListener('turbolinks:load', () => {
+  consumer.subscriptions.create("RoomChannel", {
```

Comment:
`chatChannel` が定義されていませんよ。

```suggestion
  const chatChannel = consumer.subscriptions.create("RoomChannel", {
```